maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationServiceTest.java

```diff
@@ -394,7 +398,8 @@ void givenCrownCourt_whenUpdateIsInvoked_thenApplicationDTOIsUpdatedWithNewHards
                 .thenReturn(assessmentSummaryDTO);
         when(hardshipMapper.getUserActionDTO(any(), any()))
                 .thenReturn(UserActionDTO.builder().username("mock-u").build());
-        when(applicationTrackingMapper.build(any(), any())).thenReturn(new ApplicationTrackingOutputResult().withUsn(12345));
+        when(applicationTrackingMapper.build(any(), any(), any(ApplicationTrackingOutputResult.AssessmentType.class),
+                any(ApplicationTrackingOutputResult.RequestSource.class))).thenReturn(new ApplicationTrackingOutputResult().withUsn(12345));
```

Comment:
Would be better to use a constant here.

It would also be nice to specify the actual types rather than using `any()`.